database/postgres/pg.test.js

```diff
@@ -4,7 +4,6 @@ const logger = { error: jest.fn(), info: jest.fn() };
 logGen.mockReturnValue(logger);
 
 jest.mock("pg");
-jest.mock("../../lib/log");
```

Comment:
why?

Reply:
It's on line 1, I didn't think we needed to mock it twice